README.md

```diff
@@ -47,7 +47,23 @@ Follow the steps below to compile and install Soufflé on a UNIX system:
 1.  Install the needed dependencies. G++ 4.8 or greater is recommended to compile Soufflé. openjdk-7 can be used if openjdk-8 is not available on your platform.
 
 ```
-sudo apt-get install autoconf automake bison build-essential doxygen flex g++ git libsqlite3-dev libtool make openjdk-8-jdk pkg-config python zlib1g-dev
+apt-get install -y build-essential;
```

Comment:
I've split it into two commands as the first set of packages is matched name for name in most yum based package repositories I've looked at. For example it works (replacing `apt-get` with `yum`) on Amazon Linux, which maintains compatibility with Red Hat Enterprise Linux 6. My thinking is that this provides a better starting point for people to try and install Souffle's dependencies on systems we do not provide a lot of support for. As we test on different systems, we can add further instructions for them, still with the advantage of this fall back.

 The -y should be removed, I agree it should be up to the user.